.travis.yml

```diff
@@ -20,6 +19,7 @@ cache:
     - $HOME/.cache/pyensembl/GRCh38/ensembl87/
     - $HOME/.cache/pyensembl/GRCm38/ensembl87/
     - $HOME/.cache/vaxrank
+    - $HOME/.cache/pip
```

Comment:
Do you think this changes the behavior? If so, I want to standardize this across all the repos.

Reply:
I... don't know that it does, but I want to see if it does?